ibis/sql/postgres/tests/test_functions.py

```diff
@@ -510,14 +510,14 @@ def test_numeric_builtins_work(alltypes, df):
             lambda t: (t.double_col > 20).ifelse(10, -20),
             lambda df: pd.Series(
                 np.where(df.double_col > 20, 10, -20),
-                dtype='int64'
+                dtype='int8'
```

Comment:
How does this pass?

Reply:
Because each vaule is between int8's boundaries so the expression's dtype is `dt.int8` instead of int64.